src/RA_CodeNotes.cpp

```diff
@@ -35,9 +35,9 @@ size_t CodeNotes::Load(unsigned int nID)
         if (sNote.length() < 2U)
             continue;
 
-        const std::string& sAddr { note["Address"].GetString() };
-        const auto nAddr { static_cast<ra::ByteAddress>(std::stoul(sAddr, nullptr, 16)) };
-        const std::string& sAuthor { note["User"].GetString() }; // Author?
+        const std::string& sAddr{note["Address"].GetString()};
+        const ra::ByteAddress nAddr{std::stoul(sAddr, nullptr, 16)};
```

Comment:
If the left-sides type can hold the right sides value, brace-initialization is the way to go. The `string&`s from a C-String (possible invalidation, dangling) will have to change later but tried to keep the PR focused.